website/src/pages/api/v0/announcements.ts

```diff
@@ -16,14 +16,28 @@ export default async function handle (req: NextApiRequest, res: NextApiResponse<
       'zh-Hans': '请更新至最新版本应用程式使用去中心化贷款。',
       'zh-Hant': '請更新至最新版本應用程式使用去中心化貸款。'
     },
-    version: '<0.17.0'
+    version: '<0.17.0',
```

Comment:
Can you revert this change? Even with older users, they can't still update because code for new banner is not yet existing

Reply:
then instead of removing it entirely, why not we target after the version we added announcement, until the one version before the latest version? 

I check that 0.12 is when we added announcement banner